man/systemd-sbsign.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+
+<refentry id="systemd-sbsign"
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>systemd-sbsign</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>systemd-sbsign</refentrytitle>
+    <manvolnum>1</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>systemd-sbsign</refname>
+    <refpurpose>Sign PE binaries for EFI secure boot</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>systemd-sbsign</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="req">COMMAND</arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>systemd-sbsign</command> can be used to sign PE binaries for EFI secure boot.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>Commands</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>sign</option></term>
+
+        <listitem><para>Signs the given PE binary for EFI secure boot. Takes a path to a PE binary as its
+        argument. If the PE binary already has a certificate table, the new signature will be added to it.
+        Otherwise a new certificate table will be created. Note that this command modifies the given binary
+        in place.</para>
```

Comment:
The in place by default seems like a potential footgun. An option to output a `.signed.efi` instead would be nice, otherwise this would always be dance of making a copy first.

Reply:
The convention is to create a .efi.signed yes, our tools already pick those up if present

Reply:
hmm, we should probably make this work how our other tools work. ukify as "--output=" to specify the output file, and if omitted it uses the name of the input (specifically the --linux= file) and patches in `.signed.efi`. We should probably mimic that here, I think it makes sense.

Reply:
`.efi.signed`, not `.signed.efi`

Reply:
Both of your are wrong lol, ukify just adds `.efi` if signed and `.unsigned.efi` if not.

Reply:
lol - we should separately fix ukify then, the convention is .efi.signed, bootctl uses this too

Reply:
I just made `--output` required for now to circumvent this mess.